web-registry/src/lib/db/types/json-ld/toucan-project-metadata.ts

```diff
@@ -15,6 +17,8 @@ export interface ToucanProjectMetadataLD
   'regen:approvedMethodologies': ApprovedMethodologies;
   'regen:toucanProjectTokenId': number;
   'regen:toucanURI': string;
+  'regen:rating': BeZeroRating;
+  'regen:certification': Certification;
```

Comment:
I believe the certification is meant to be stored at the credit class level, see issue description https://github.com/regen-network/regen-registry/issues/1710

Reply:
You're right, will fix it.

Reply:
I moved `regen:certification` to the credit class metadata: [bca7d5c](https://github.com/regen-network/regen-web/pull/1942/commits/bca7d5c74174c981ac80b76fc66611ef44382ad8)
I moved it to `CreditClassMetadataLD` as an optional field. Should we be more explicit and put it as a mandatory field in `CFCCreditClassMetadataLD` and `ToucanCreditClassMetadataLD` (which do not exist currently)?